library/Jira/Common/IcingaDbSupport.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+// Icinga Web Jira Integration | (c) 2024 Icinga GmbH | GPLv2
+
+namespace Icinga\Module\Jira\Common;
+
+use Icinga\Application\Icinga;
+use Icinga\Application\Modules\Module;
+use Icinga\Authentication\Auth;
+
+trait IcingaDbSupport
```

Comment:
Use the existing class instead, please. You can overwrite the method if required.

Reply:
Using the existing class sounds good. There is no need to rewrite the method.